lib/api/apiUtils/object/createAndStoreObject.js

```diff
@@ -148,6 +148,7 @@ function createAndStoreObject(bucketName, bucketMD, objectKey, objMD, authInfo,
             removeAWSChunked(request.headers['content-encoding']);
         metadataStoreParams.expires = request.headers.expires;
         metadataStoreParams.tagging = request.headers['x-amz-tagging'];
+        metadataStoreParams.originOp = 's3:ObjectCreated:Put';
```

Comment:
I am ok with the string literal here, but are you going to maintain a dictionary/list of ops?

Reply:
There's a list being added in the Arsenal pr for putBucketNotification